packages/app-store/routing-forms/trpc/_router.ts

```diff
@@ -98,12 +98,12 @@ const appRoutingForms = router({
       return handler({ ctx, input });
     }),
 
-  findTeamMembersMatchingAttributeLogic: authedProcedure
+  findTeamMembersMatchingAttributeLogicOfRoute: authedProcedure
```

Comment:
As explained earlier it is a rename to convey its distinction from similar trpc route.